inc/saiport.h

```diff
@@ -219,12 +219,18 @@ typedef enum _sai_port_interface_type_t
     /** Interface type CR */
     SAI_PORT_INTERFACE_TYPE_CR,
 
+    /** Interface type CR2 */
+    SAI_PORT_INTERFACE_TYPE_CR2,
+
     /** Interface type CR4 */
     SAI_PORT_INTERFACE_TYPE_CR4,
 
     /** Interface type SR */
     SAI_PORT_INTERFACE_TYPE_SR,
 
+    /** Interface type SR2 */
+    SAI_PORT_INTERFACE_TYPE_SR2,
```

Comment:
can you remove this from the pr, it is not related to udf.

Reply:
There was an issue with signoff. Fixed it